controllers/operandrequest/reconcile_operator.go

```diff
@@ -160,6 +160,22 @@ func (r *Reconciler) reconcileSubscription(ctx context.Context, requestInstance
 	namespace := r.GetOperatorNamespace(opt.InstallMode, opt.Namespace)
 	sub, err := r.GetSubscription(ctx, opt.Name, namespace, registryInstance.Namespace, opt.PackageName)
 
+	if opt.UserManaged {
+		klog.Infof("Skip installing operator %s because it is managed by user", opt.PackageName)
+		if sub == nil {
+			return errors.New("operator" + opt.Name + " is user managed, but no Subscription exists, waiting...")
+		}
+		sub.Annotations[requestInstance.Namespace+"."+requestInstance.Name+"."+operand.Name+"/request"] = opt.Channel
+		sub.Annotations[requestInstance.Namespace+"."+requestInstance.Name+"."+operand.Name+"/operatorNamespace"] = namespace
+		if err = r.updateSubscription(ctx, requestInstance, sub); err != nil {
+			requestInstance.SetMemberStatus(opt.Name, operatorv1alpha1.OperatorFailed, "", mu)
+			return err
+		}
+		requestInstance.SetMemberStatus(opt.Name, operatorv1alpha1.OperatorUpdating, "", mu)
+		// requestInstance.SetMemberStatus(operand.Name, operatorv1alpha1.OperatorRunning, operatorv1alpha1.ServiceRunning, mu)
+		return nil
+	}
+
```

Comment:
Hi @bitscuit 

I am wondering if we are able to handle the case where user creates subscription in `openshift-operators` namespace with all namespaces mode, and install CPFS in own namespace mode.

In this case, we will not seeing any subscription in CPFS tenant.

Reply:
I am thinking it is difficult to handle the uninstallation at this moment since most of our uninstallation is based on subscription annotations.

I feel that if we could not solve it in 4.6.6 release, we might consider it as known issue on uninstallation when `user managed` feature is on.

Reply:
I am facing the same uninstallation with fallback channels, I am seeing if I could refactor the logic in uninstallation section.

Reply:
done